test/common_test/world_features.cc

```diff
@@ -457,6 +457,56 @@ TEST_F(WorldNestedModelTest, WorldConstructNestedModel)
   }
 }
 
+struct WorldSolverFeatureList : gz::physics::FeatureList<
+  gz::physics::GetEngineInfo,
+  gz::physics::Solver,
+  gz::physics::sdf::ConstructSdfWorld
+> { };
+
+
+class WorldSolverTest : public WorldFeaturesTest<WorldSolverFeatureList>
+{
+  public: gz::physics::World3dPtr<WorldSolverFeatureList> LoadWorld(
+      const std::string &_pluginName)
+  {
+    gz::plugin::PluginPtr plugin = this->loader.Instantiate(_pluginName);
+
+    auto engine =
+        gz::physics::RequestEngine3d<WorldSolverFeatureList>::From(plugin);
+
+    sdf::Root root;
+    const sdf::Errors errors = root.Load(
```

Comment:
include for `sdf::Errors`

Reply:
added. [adc1b3d](https://github.com/gazebosim/gz-physics/pull/609/commits/adc1b3dac83b67461cf0a67c8566540956782307)